external/jenkins/README.md

```diff
@@ -0,0 +1,21 @@
+## External Jenkins Tests
+
+Jenkins tests are part of the external third-party application tests that help verify that the AdoptOpenJDK binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
+
```

Comment:
Please add the extra sentence about where the Jenkins test material comes from

Reply:
@smlambert Actually I have added it at the end of the file but will move it to above as you state